front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SurveyQuestionResultWidget/Question/index.tsx

```diff
@@ -50,69 +50,90 @@ const SurveyQuestionResult = ({
   const localize = useLocalize();
   const { formatMessage } = useIntl();
 
+  const renderQuestionComponent = useCallback(() => {
+    if (!data) return null;
+
+    const { attributes } = data.data;
+
+    switch (attributes.inputType) {
+      case 'ranking':
```

Comment:
Wanted to clean up the existing conditional statements, let me know if there's a better solution though :)